CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ appear at the top.
 
   * Your contribution here!
 
+## [1.17.0][] (2018-06-08)
+
+  * [#430](https://github.com/capistrano/sshkit/pull/430): [Feature] Command Argument STDOUT/capistrano.log Hiding - [@NorseGaud](https://github.com/NorseGaud)
```

Comment:
Please move this to the Unreleased section (next to the "Your contribution here!" bullet).